modules/nw-multus-advanced-annotations.adoc

```diff
@@ -0,0 +1,202 @@
+= Specifying pod-specific addressing and routing options
+
+When attaching a pod to an additional network, you may wish to specify further properties
+about that network in a particular pod. This allows you to change some aspects of routing, as well
+as specify static IP addresses and MAC addresses. In order to accomplish this, you may use the JSON formatted annotations.
+
+.Prerequisites
+
+* The Pod must be in the same namespace as the additional network.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster.
+ifdef::sriov[]
+* You must have the SR-IOV Operator installed and a SriovNetwork CR defined.
+endif::sriov[]
+
+.Procedure
+
+To add a Pod to an additional network while specifying addressing and/or routing options, complete the following steps:
+
+. Edit the Pod resource definition. If you are editing an existing Pod, run the
+following command to edit its definition in the default editor. Replace `<name>`
+with the name of the Pod to edit.
++
+----
+$ oc edit pod <name>
+----
+
+. In the Pod resource definition, add the `k8s.v1.cni.cncf.io/networks`
+parameter to the Pod `metadata` mapping. The `k8s.v1.cni.cncf.io/networks`
+accepts a JSON string of a list of objects that reference the name of NetworkAttachmentDefinition Custom Resource (CR) names
+in addition to specifying additional properties.
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[<network>[,<network>,...]]' <1>
+----
+<1> Replace `<network>` with a JSON object as shown in the following examples. The single quotes are required.
+
+. In the following example the annotation specifies which network attachment will have the default route,
+using the `default-route` paramter.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '
+    {
+      "name": "net1"
+    },
+    {
+      "name": "net2", <1>
+      "default-route": ["192.0.2.1"] <2>
+    }'
+spec:
+  containers:
+  - name: example-pod
+    command: ["/bin/bash", "-c", "sleep 2000000000000"]
+    image: centos/tools
+----
+<1> The `name` key is the name of the additional network to associate
+with the Pod.
+<2> The `default-route` key specifies a value of a gateway for traffic to be routed over if no other
+routing entry is present in the routing table. If more than one `default-route` key is specified,
+this will cause the pod to fail to become active.
+
+The default route will cause any traffic that is not specified in other routes to be routed to the gateway.
+
+[IMPORTANT]
+====
+Setting the default route to an interface other than the default network interface for {product-title}
+may cause traffic that is anticipated for pod-to-pod traffic to be routed over another interface.
+====
+
+To verify the routing properties of a pod, the `oc` command may be used to execute the `ip` command within a pod.
++
```

Comment:
I don't think you need this "+" and it's showing up in the preview build. Just remove it.